src/styles/blocks/card.scss

```diff
@@ -0,0 +1,77 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
```

Comment:
Consider using a variable for the border color (`#f3f3f3`) to maintain consistency and ease future changes. This will also improve maintainability if the color is used elsewhere.